docs/abs.md

```diff
@@ -0,0 +1,77 @@
+# Always be Scheduling (ABS)
+
+**NOTE: This facility is not publically available and can only be used by
+personnel employed by Puppet the company.**
+
+[Always be Scheduling](https://github.com/puppetlabs/always-be-scheduling)
+is a host scheduler to ensure that hosts are returned to the requester as
+quickly as possible.  Requests are queued in order to fill them on in the
```

Comment:
We may want to note that we use the `awsdirect` endpoint which "allows you to make a direct request for an aws resource instead of the request going through the queue-processor".

Reply:
Just thinking about the new docs directory here: If we are going to create a new docs directory should any of the other docs (CD4PE.md, etc...) be moved there for consistency? Perhaps even move the NEW_README.md there and rename it to something more descriptive? 

Also, the names of the .md files usually seem to be capitalized. Is this actually a standard? It makes sense for the README.md file to make it stand out but not sure if it is necessary if we start a docs directory. However, we should probably adhere to a standard within a directory for consistency.

Reply:
@billclaytor I have opened [SLV-356](https://tickets.puppetlabs.com/browse/SLV-356) to address the organization issue.

Reply:
@johnduarte Thanks!